corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -203,6 +205,9 @@ hqDefine("app_manager/js/details/case_claim", function () {
         };
 
         subscribeToSave(self, searchConfigKeys, saveButton);
+        $(".case-search-multimedia-input button").on("click", function () {
```

Comment:
Is this how other multimedia in app manager handles firing save button changes?

Reply:
I used the case list implementation as the reference for this feature request,
https://github.com/dimagi/commcare-hq/blob/e98ee689ae5356bab3f3e5bbdb2badabfc99fd40/corehq/apps/app_manager/static/app_manager/js/details/case_list_callout.js#L21

Reply:
cool, thanks for the link